lib/matplotlib/figure.py

```diff
@@ -958,10 +958,11 @@ def legend(self, *args, **kwargs):
         Call signatures::
 
             legend()
-            legend(labels)
             legend(handles, labels)
+            legend(handles=handles)
+            legend(labels)
 
-        The call signatures correspond to these three different ways to use
+        The call signatures correspond to these four different ways to use
```

Comment:
```suggestion
        The call signatures correspond to these different ways to use
```
if you don't have a number, it doesn't need to be updated every time the API changes.

Reply:
Slight tweak, then:
```suggestion
        The call signatures correspond to the following ways to use
```